contracts/settling_game/L06_Combat.cairo

```diff
@@ -131,10 +131,9 @@ const MAX_WALL_DEFENSE_HIT_POINTS = 5
 
 @external
 func initializer{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
-    address_of_controller : felt, xoroshiro_addr : felt, proxy_admin : felt
```

Comment:
was planning on moving this to a setter rather than in the initializer. I don't think it needs to be here. Let's just set a function that gets called - or even better - let's store it in the module controller so it can be accessed from other modules.

Reply:
Trying to make all modules have the same constructor args, which will enable people to deploy versions of the game easily